optimum_benchmark/utils.py

```diff
@@ -148,7 +139,8 @@ def check_only_this_process_is_running_on_cuda_device(
                 ]
             )
 
-            # check if there is a process running on device_id that is not the current process
+            # check if there is a process running on
+            # device_id that is not the current process
```

Comment:
I think PEP-8 80 characters per line is too strict. It could make sense to add a config like https://github.com/huggingface/optimum/blob/main/pyproject.toml#L19-L23

Reply:
done !